internal/commands/root/root.go

```diff
@@ -42,6 +42,13 @@ import (
 	"k8s.io/client-go/tools/record"
 )
 
+var (
+	// podController is the pod controller which created by NewPodController.
+	podController *node.PodController
```

Comment:
We cannot use global vars like this.
These need to be plumbed through from whatever is returned from `NewCommand`.

If that means we need to return extra things or wrap `*cobra.Command` to add accessors, that's fine.

Reply:
Ok... review again.
@cpuguy83